### PR TITLE
Api-Login and Profile

### PR DESCRIPTION
**Pull request by Krutikaa04**
created two APIs -->
POST login api
GET profile api

database collection named users
sample data included

jwt token generated.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+require("dotenv").config();
```
Comment (shahkushal38) [inline app.js]: This line should come after const jwt line.... ie line 6

File: app.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+//login api started
+
+app.post('/login', async (req, res)=>{
+
+  const { username, password, email, branch, tokens} = req.body;
+  let token;
+
+  if(!username || !password || !email || !branch){
+    return res.status(422).json({error:"fill field properly"});
```
Comment (shahkushal38) [inline app.js]: Why response status as 422? Isn't it be 400 

And check for all other cases too please

File: app.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    if(userExist){
+        return res.status(422).json({error: "Email already exists"});
+    }
+    
+
+    const user = new Users({username, password, email, branch});
+
+
+    //generate token
+    token = jwt.sign({_id: this._id, email: this.email}, process.env.SECRET_TOKEN, {expiresIn: "30s"});
```
Comment (shahkushal38) [inline app.js]: This should not be 30s it should be atleast 24hrs

File: model/maindb.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+
+
+
+
+
+
+
+
+// function userInsert(){
```
Comment (shahkushal38) [inline model/maindb.js]: remove this unwanted code if not needed

File: model/schema1.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
```
Comment (shahkushal38) [inline model/schema1.js]: Please rename the file to userSchema.js to make it understandable

**Review by shahkushal38 — CHANGES_REQUESTED** [review]
LGTM! Have given suggestions to make the code clean and better to understand

**Comment by Krutikaa04** [comment]
Resolved the following : 
1.  status codes changed accordingly
2.  jwt expiry time changed from 30s to 24h
3.  removed unwanted code
4. renamed the file 
5. the database url and the port are now hidden

**Review by shahkushal38 — APPROVED** [review]
LGTM! PR Approved

**Review by EshantBagwe — APPROVED** [review]

**Review by shahkushal38 — APPROVED** [review]
approved

**Pull request merged**